keycloak/keycloak-operator
# Intly 3612

## JIRA ID
https://issues.jboss.org/browse/INTLY-3611
https://issues.jboss.org/browse/INTLY-3612
https://issues.jboss.org/browse/INTLY-3613

## Additional Information
Adding the monitoring resource creation to the operator. Since these resource types may not exist on the cluster, this needs to be checked on the cluster before setting up a watch and before reconciling the resources

## Verification Steps
1. Checkout this branch locally
2. Run `make cluster/prepare`
3. Run the operator and watch the logs
4. Run `make cluster/create/examples`
5. Verify the output of the logs does not create any monitoring resources
6. clone this repo - https://github.com/integr8ly/application-monitoring-operator
7. Run `make cluster/install`
8. Verify the operator logs output the watches are setup
9. Verify the 3 monitoring resources have been created.

## Checklist:

## Additional Notes
